package.json

```diff
@@ -24,7 +24,9 @@
     "testcafe": "testcafe chrome tests/ -s \"tests/eyes/screenshots\"",
     "precommit": "yarn format:fix",
     "format:fix": "pretty-quick --staged",
-    "format:check": "prettier --config ./.prettierrc --list-different \"src/app/**/*{.ts, .scss}\""
+    "format:check": "prettier --config ./.prettierrc --list-different \"src/app/**/*{.ts, .scss}\"",
+    "dorit": "testcafe chrome tests/student-schedule.e2e.ts  --selector-timeout 10000 --speed 1 -r json > test.txt",
```

Comment:
Can you please change name or remove these lines that are dedicated on your name?
Thanks